### PR TITLE
Improve the UI of skill card

### LINKED ISSUE #1710: Improve UI of skill card
**Actual Behaviour**

Skills title are not in center

**Expected Behaviour**

Skills title should be in center and the skill card UI can be improved

**Screenshots of the issue**

screenshot_2018-09-26-17-11-22-089_ai susi

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by GOVINDDIXIT**
Fixes #1710 

## Changes done in PR
1. Title of skill is made center alligned.
2. A separate space is given to rating bar so that in future average rating can also be shown with each 
    skill.
3. Separate space is given to total ratings also

Screenshots for the change: 
Before:
screenshot_2018-09-26-17-11-22-089_ai susi

After:
screenshot_2018-09-26-17-21-52-953_ai susi

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Please list the changes done in PR description

**Comment by GOVINDDIXIT** [comment]
@iamareebjamal  pls review

File: app/src/main/res/layout/item_skill.xml [hunk 1]
```diff
@@ -77,7 +78,7 @@
                 android:textSize="@dimen/text_size_moderate"
                 android:fontFamily="sans-serif"/>
 
-            <LinearLayout
+            <RelativeLayout
```
[the remark this answers is not in the document]
Comment (GOVINDDIXIT) [reply app/src/main/res/layout/item_skill.xml]: Indeed senpai 😅

File: app/src/main/res/layout/item_skill.xml [hunk 2]
```diff
@@ -77,26 +78,40 @@
...
 
                 <RatingBar
                     android:id="@+id/cv_rating_bar"
                     style="?android:attr/ratingBarStyleSmall"
                     android:layout_width="@dimen/rating_bar_layout_width"
                     android:layout_height="wrap_content"
+                    android:layout_gravity="center"
                     android:numStars="5"
                     android:padding="@dimen/cmv_padding"
                     android:theme="@style/RatingBar" />
```
Comment (iamareebjamal) [inline app/src/main/res/layout/item_skill.xml]: Indentation of the RatingBar is wrong

Comment (GOVINDDIXIT) [reply app/src/main/res/layout/item_skill.xml]: So it should be left aligned?

Comment (iamareebjamal) [reply app/src/main/res/layout/item_skill.xml]: It should be one indent less. Please correct me if I am wrong

Comment (GOVINDDIXIT) [reply app/src/main/res/layout/item_skill.xml]: It is looking fine for now, can be shifted to left when average stars will be shown along with rating. Pls give your views.

**Comment by GOVINDDIXIT** [comment]
@iamareebjamal pls review

File: app/src/main/res/layout/item_skill.xml [hunk 3]
```diff
@@ -77,26 +78,40 @@
                 android:textSize="@dimen/text_size_moderate"
                 android:fontFamily="sans-serif"/>
 
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="horizontal">
 
                 <RatingBar
```
[the remark this answers is not in the document]
Comment (GOVINDDIXIT) [reply app/src/main/res/layout/item_skill.xml]: Oh 😱 I have forgot to reformat the code, To the whole time I was thinking that you were asking me to shift the rating bar to the left after rendering.

[the remark this answers is not in the document]
Comment (GOVINDDIXIT) [reply app/src/main/res/layout/item_skill.xml]: pls review😅

File: app/src/main/res/layout/item_skill.xml [hunk 4]
```diff
@@ -51,15 +51,15 @@
...
                     android:gravity="center"
                     android:maxLines="3"
                     android:padding="@dimen/padding_small"
                     android:textAlignment="center"
                     android:textColor="@color/md_grey_700"
                     android:textSize="@dimen/text_size_normal"
                     android:textStyle="italic"
-                    tools:text="@string/sample_website_description"
-                    android:fontFamily="sans-serif"/>
+                    tools:text="@string/sample_website_description" />
```
Comment (codedsun) [inline app/src/main/res/layout/item_skill.xml]: unrelated change here, you just have moved the lines from one position to another

Comment (GOVINDDIXIT) [reply app/src/main/res/layout/item_skill.xml]: The actual work was already done in earlier commits which is mentioned in pr description but I forget to reformat the code so I used Ctrl + Alt + L

**Comment by arundhati24** [comment]
@GOVINDDIXIT Your code changes look fine, but I am sorry I had been a bit occupied with some work due to which I could not check your issue/PR. Actually, the UI changes have to be consistent across different clients and we do not require the changes that you have made here. @jogendra @anshumanv What do you think?

**Comment by anshumanv** [comment]
Makes sense @arundhati24. :+1:

**Comment by jogendra** [comment]
@GOVINDDIXIT Thank you for making PR, great work but the UI changes you made is not according to our guideline.

**Comment by GOVINDDIXIT** [comment]
Ok No problem 😃, what should I do now.
